master/reference/calicoctl/commands/node/run.md

```diff
@@ -104,6 +130,83 @@ CALICO_LIBNETWORK_ENABLED is true - start libnetwork service
 Calico node started successfully
 ```
 
+#### IP Autodetection method examples
+
+The node resource includes IPv4 and IPv6 routing IP addresses that should
+match those on one of the host interfaces.  These IP addresses may be 
+configured in advance by configuring the node resource prior to starting the
+calico/node service, or alternatively the addresses may either be explicitly 
```

Comment:
`service, or alternatively the addresses` --> `service, alternatively, the addresses` ? I  think?